docker/Dockerfile

```diff
@@ -16,9 +16,13 @@ RUN curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | REPO_S
     && rm -rf /var/lib/apt/lists/* /tmp/* \
     && apt-get autoremove \
     && find /var/log -name "*.log" -type f -delete \
-    && sed -i "s/num_threads 8/num_threads 8\n  detect_json true\n  # Enable metadata agent lookups.\n  enable_metadata_agent true\n  metadata_agent_url \"http:\/\/local-metadata-agent.stackdriver.com:8000\"/" "/etc/google-fluentd/google-fluentd.conf"
+    && sed -i "s/num_threads 8/num_threads 8\n  detect_json true\n  # Enable metadata agent lookups.\n  enable_metadata_agent true\n  metadata_agent_url \"http:\/\/local-metadata-agent.stackdriver.com:8000\"/" "/etc/google-fluentd/google-fluentd.conf" \
+    # Pin systemd-journal to < 1.3.2 until https://github.com/ledbettj/systemd-journal/issues/79 is fixed.
```

Comment:
Do we want to run this command right after installing the logging agent (e.g., before we delete the logs, etc)?

Reply:
Oops. I force rebased before seeing this comment (thought no one had seen this PR yet).

I abandoned this approach and pinned the version in `plugin_gems.rb` instead (last time we were in a rush)